eng/Version.Details.xml

```diff
@@ -122,6 +122,10 @@
       <Uri>https://github.com/dotnet/runtime-assets</Uri>
       <Sha>b34a70799faa67f13c2e72852e4f3af463ff4d6c</Sha>
     </Dependency>
+    <Dependency Name="Microsoft.NETCore.Runtime.ICU.Transport" Version="5.0.0-preview.8.20364.2">
+      <Uri>https://github.com/dotnet/icu</Uri>
+      <Sha>d75e68181dfd8cad98dfd1c42c3ac2f893690d66</Sha>
+    </Dependency>
```

Comment:
this was moved to the ProductDependencies node above, so it shouldn't show up here. Git merge didn't see it I guess?